src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportWizard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { PageHeader, Space, StepsProps } from "antd";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+
+interface SampleMetadataImportWizardProps {
+  current: StepsProps["current"];
+  status?: StepsProps["status"];
+  percent?: StepsProps["percent"];
+  children: React.ReactNode;
+}
+
+/**
+ * React component that displays the Sample Metadata Uploader Wizard wrapper.
+ * @prop current - the current step, starting with zero
+ * @prop status - the status of the current step
+ * @prop percent - the progress percentage of the current step
+ * @prop children - the status of the current step
+ * @returns {*}
+ * @constructor
+ */
+export function SampleMetadataImportWizard({
+  current,
+  status,
+  percent,
+  children,
+}: SampleMetadataImportWizardProps): JSX.Element {
+  const { projectId } = useParams<{ projectId: string }>();
+
+  return (
+    <PageHeader
+      title={i18n("SampleMetadataImportWizard.title")}
+      subTitle={i18n("SampleMetadataImportWizard.intro")}
+      onBack={() =>
+        (window.location.href = setBaseUrl(`projects/${projectId}/linelist`))
+      }
+    >
+      <Space direction="vertical" size="large" style={{ width: `100%` }}>
+        <SampleMetadataImportSteps
+          current={current}
+          status={status}
+          percent={percent}
+        />
+        {children}
```

Comment:
I still don't like how this is set up.  A better way to handle this would be to use this as a layout file and replace the `{children}` with the react-router `Outlet` component.

Reply:
You already approved this. We worked on it together, because you didn't like how the page "flickered".

Reply:
I have to admit I do not remember working it through with you (possibly because I am old).  I just spent a couple of minutes playing with it and do not see any "flickering", the only issue is with the steps and the data you are passing into it.  This could be fixed by creating a React context to hold that data and this is probably the better way to do it.  The reason I would like to see this update is so that the header does not need to be re-rendered on each step.  Even though this is not a big update, can you please do it in a separate PR?

Reply:
@joshsadam can you write this up as an issue

Reply:
Sure can.  Issue created: #1456